src/Service/CloudFormation/src/Result/StackEvent.php

```diff
@@ -43,6 +45,8 @@ class StackEvent
 
     /**
      * Current status of the resource.
+     *
+     * @see ResourceStatus
```

Comment:
does somebody know how to define enum in dockblock? something like `@var string{'foo'|'bar'}`.

Reply:
Look at the bottom of this page: https://psalm.dev/docs/annotating_code/typing_in_psalm/